jdbc-core/src/main/java/com/salesforce/datacloud/jdbc/core/DataCloudStatement.java

```diff
@@ -15,16 +15,11 @@
  */
 package com.salesforce.datacloud.jdbc.core;
 
-import static com.salesforce.datacloud.jdbc.core.HyperGrpcClientExecutor.HYPER_MAX_ROW_LIMIT_BYTE_SIZE;
-import static com.salesforce.datacloud.jdbc.core.HyperGrpcClientExecutor.HYPER_MIN_ROW_LIMIT_BYTE_SIZE;
```

Comment:
why were those moved here? Layering-wise, they felt more natural in the `HyperGrpcClientExecutor` to me

Reply:
Given that this is not a regression, feel free to ship this PR as is - or fix in the same PR. Just let me know what you prefer to do :)